### PR TITLE
Added Unit Tests for Baseline Submission Leaderboard and Toggle

### PR DESCRIPTION
**Pull request by vkartik97**

### CODE REVIEW

**Comment by vkartik97** [comment]
@RishabhJain2018 @deshraj please review!

File: tests/unit/jobs/test_views.py [hunk 1]
```diff
@@ -1720,6 +1870,143 @@ def test_get_leaderboard(self):
...
+                    "submission__submitted_at": self.submission.submitted_at,
+                    "submission__is_baseline": False,
+                    "submission__method_name": self.submission.method_name,
+                }
+            ],
+        }
+        expected = collections.OrderedDict(expected)
+        response = self.client.get(self.url, {})
+
+        # Teardown
```
Comment (RishabhJain2018) [inline tests/unit/jobs/test_views.py]: What do you mean by this comment?

File: tests/unit/jobs/test_views.py [hunk 2]
```diff
@@ -1720,6 +1870,143 @@ def test_get_leaderboard(self):
...
+                    "submission__is_baseline": True,
+                    "submission__method_name": self.host_participant_team_submission_2.method_name,
+                }
+            ],
+        }
+        expected = collections.OrderedDict(expected)
+        response = self.client.get(self.url, {})
+        print(expected)
+        print(response.data)
+        # Teardown
```
Comment (RishabhJain2018) [inline tests/unit/jobs/test_views.py]: Same as above.

Comment (vkartik97) [reply tests/unit/jobs/test_views.py]: teardown are the steps that are excited after this test has executed

**Comment by vkartik97** [comment]
@RishabhJain2018 @rishabhbhardwaj Please review

**Comment by rishabhbhardwaj** [comment]
@RishabhJain2018 changes look good to me

File: tests/unit/jobs/test_views.py (hunk 1 above)
Comment (RishabhJain2018) [reply tests/unit/jobs/test_views.py]: :)

**Pull request merged**